recipes/xrandr.lua

```diff
@@ -131,7 +130,6 @@ local function xrandr()
                               function()
                                  state.timer:stop()
                                  state.timer = nil
-                                 state.iterator = nil
```

Comment:
I was thinking of:
- Use this to change the current screen configuration
- Wait 5 hours and do other work
- Connect a new monitor to the computer
- Try to enable it with this module.

Since the "menu" is not rebuilt at this point, you still only get the old possibilities. Same problem when a monitor is disconnected.

I do not really have a preference between "restart iteration at the beginning" and "restart iteration where it was stopped last time", but the above might be a problem.